styles/base/form.scss

```diff
@@ -93,28 +80,25 @@ label {
     z-index: 2;
     float: left;
     width: 100%;
+    border-right: 0;
+    border-left: 0;
+
+    &:focus { 
+      box-shadow: 0 0 2px 1px rgba(82,181,229,1);
```

Comment:
Yea, but we still follow our design. If you want to add shadows, we need to be consistent. This still needs discussion, so.. :).